app/frontend/wallet/address-manager.ts

```diff
@@ -59,10 +67,11 @@ const AddressManager = ({addressProvider, gapLimit, blockchainExplorer}) => {
     })
   }
 
-  function getAddressToAbsPathMapping() {
+  // this is supposed to return {[key: _Address]: BIP32Path} but ts does support
```

Comment:
I think this is a feature rather than a "bug" given that arbitrary types may not be hashable/comparable in a way that would make the mapping work as expected, so I would not put here the comment. Maybe a type alias "AddressString" could do to make the intent clear instead of a comment

Reply:
an alternative would be returning a `Map` which I'd guess typescript would allow to have arbitrary entries as that's what's in the specs: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map - but I can see it having unintended side effects if we refactor the _Address type to actually be an object so I'd rather not go down that path